src/transaction/log_page_buffer.c

```diff
@@ -7333,11 +7326,6 @@ logpb_checkpoint (THREAD_ENTRY * thread_p)
 
   LOG_CS_EXIT (thread_p);
 
-#if 0
-  /* have to sync log vol, data vol */
-  fileio_synchronize_all (thread_p, true /* include_log */ );
```

Comment:
I'm not sure if it has to be done or not though it's not done now. It could be a temporary fix..